Cubical/Algebra/SymmetricGroup.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Algebra.SymmetricGroup where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Sigma
+open import Cubical.Data.Nat using (ℕ ; suc ; zero)
+open import Cubical.Data.Fin using (Fin ; isSetFin)
+open import Cubical.Data.Empty
+open import Cubical.Relation.Nullary using (¬_)
+
+open import Cubical.Structures.Group
+
+private
+  variable
+    ℓ ℓ' : Level
+
+infixr 20 _∘_
+
+_∘_ : {A B C : Type ℓ} → B ≃ C → A ≃ B → A ≃ C
+g ∘ f = compEquiv f g
+
+∘-assoc : {A B C D : Type ℓ} (h : C ≃ D) (g : B ≃ C) (f : A ≃ B)
```

Comment:
Can you upstream this result as well? I don't think we need the notation for compEquiv as we already use it for function composition

Reply:
Okay, I will move the result and eliminate the notation.

Reply:
Great!